modules/consensus/persist.go

```diff
@@ -47,14 +59,20 @@ func (cs *ConsensusSet) load(saveDir string) error {
 		return cs.load(saveDir)
 	}
 
+	// The state cannot be easily reverted to a point where the
+	// consensusSetHash can be re-made. Load from disk instead
+	pb, err := cs.db.getBlockMap(bid)
```

Comment:
`pb` is a name that breaks conventions because there's no immediate context to tell you what it stands for. This will not be a problem after renaming `getBlockMap` to `getProcessedBlock`.

So I guess it's okay for this pr